src/tests/functional/shared_test_classes/src/subgraph/memory_LSTMCell.cpp

```diff
@@ -2,332 +2,343 @@
+
+    auto cell_memory_constant = ngraph::builder::makeConstant<float>(element_type, cell_memory_dims, cell_memory_init);
+
+    auto hidden_memory_constant =
+        ngraph::builder::makeConstant<float>(element_type, hidden_memory_dims, hidden_memory_init);
+
+    // Body - layers
+    auto squeeze_const = std::make_shared<ov::op::v0::Constant>(element::i64, Shape{1}, squeeze_axes);
+    auto squeeze = std::make_shared<ov::op::v0::Squeeze>(unsqueeze_input, squeeze_const);
+
+    auto weightsNode = ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize, inputSize}, weights_vals);
+    auto reccurrenceWeightsNode =
+        ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize, hiddenSize}, reccurrenceWeights_vals);
+    auto biasNode = ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize}, bias_vals);
+    auto lstm = std::make_shared<ov::op::v0::LSTMCell>(squeeze,
+                                                       hidden_memory_constant,
+                                                       cell_memory_constant,
+                                                       weightsNode,
+                                                       reccurrenceWeightsNode,
+                                                       biasNode,
+                                                       hiddenSize);
+
+    auto unsqueeze_const = std::make_shared<ov::op::v0::Constant>(element::i64, Shape{1}, squeeze_axes);
+    auto unsqueeze = std::make_shared<ov::op::v0::Unsqueeze>(lstm->output(0), unsqueeze_const);
+
+    auto final_reshape_pattern =
+        std::make_shared<ov::op::v0::Constant>(element::i64, Shape{4}, std::vector<size_t>({1, 1, 1, hiddenSize}));
+    auto final_reshape = std::make_shared<ov::op::v1::Reshape>(unsqueeze, final_reshape_pattern, false);
+
+    function = std::make_shared<Model>(final_reshape, input_parameter, "TI_unrolled_without_memory");
+}
+
+void MemoryLSTMCellTest::create_pure_tensor_iterator_model() {
+    ov::element::Type element_type;
+    ov::AnyMap config;
+    size_t inputSize;
+    std::tie(transformation, targetDevice, element_type, inputSize, hiddenSize, config) = this->GetParam();
+
+    std::vector<size_t> input_dims{1, inputSize};
+    std::vector<size_t> squeeze_axes{0};
+    std::vector<size_t> hidden_memory_dims{1, hiddenSize};
+    std::vector<size_t> cell_memory_dims{1, hiddenSize};
+
+    ov::ParameterVector input_parameter{std::make_shared<ov::op::v0::Parameter>(element_type, ov::Shape(input_dims))};
+
+    auto input_add_const = ngraph::builder::makeConstant(element_type, input_dims, input_bias);
+    auto add = ngraph::builder::makeEltwise(input_parameter[0], input_add_const, ov::test::utils::EltwiseTypes::ADD);
+
+    auto input_mul_const = ngraph::builder::makeConstant(element_type, input_dims, input_weights);
+    auto mul = ngraph::builder::makeEltwise(add, input_mul_const, ov::test::utils::EltwiseTypes::MULTIPLY);
```

Comment:
`ngraph::builder` can be replaced by `ov` version?
Is used in many places in this file. Will it be migrated?